doc/cephadm/host-management.rst

```diff
@@ -228,6 +228,83 @@ create a new CRUSH host located in the specified hierarchy.
 
 See also :ref:`crush_map_default_types`.
 
+OS Tuning Profiles
+==================
+
+Cephadm supports managing os tuning profiles that allow users to apply a set of
+sysctl settings to a given set of hosts. To do this, first create a yaml spec
+of the following format
+
+.. code-block:: yaml
+
+    profile_name: 23-mon-host-profile
+    placement:
+      hosts:
+        - mon-host-01
+        - mon-host-02
+    settings:
+      fs.file-max: 1000000
+      vm.swappiness: '13'
+
+Then apply the tuned profile with::
+
+    ceph orch tuned-profile apply -i <tuned-profile-file-name>
+
+This profile will then be written to /etc/sysctl.d/ on each host matching the
+given placement and `sysctl --system` will be run on the host.
+
+.. note:: 
+
+  The file name the tuned profile will be written under is <profile-name>-cephadm-tuned-profile.conf.
+  Since the settings are applied in lexicographical order, you may want to set the profile name
+  for your spec in a way that is applied before or after other conf files that may exist.
+
+
+Viewing Profiles
+----------------
+
+To view all current profiles cephadm is managing::
+
+    ceph orch tuned-profile ls
+
+.. note:: 
+
+  If you'd like to make modifications and re-apply a profile passing `--format yaml` to the
+  tuned-profile ls command will present the profiles in a format where they can be copied
+  and re-applied.
+
+
+Removing Profiles
+-----------------
+
+If you no longer want one of the previously applied profiles, it can be removed with::
+
+    ceph orch tuned-profile rm <profile-name>
+
+When a profile is removed, cephadm will clean up the file previously written to /etc/sysctl.d
+
+
+Modifying Profiles
+------------------
+
+While you can modify a profile by simply re-applying a yml spec with the same profile name,
+you may also want to make a minor adjustment to a setting within a given profile commands
+have explicitly been added for this purpose
+
```

Comment:
Suggest: 

you may also want to  adjust  a setting within a given profile, so there are commands
for this purpose.